src/components/searchresult/SingleSearchResult_fof.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+    <div class="searchResult" :id="$_fixIdForSearchResultContainer(result.groupValue)">
+      <div class="generalInfo">
+        <div class="overallInfo">
+             <single-search-result-info-base :result="result"  />
+          <div class="resultInfo">Time</div>
+          <div class="timeTitle" />
+          <div class="resultDate">
+            Rapport from approx <span>{{ this.$_transformDate(result.doclist.docs[0].ctime) }} </span>
+            years ago, {{ this.$_deliverTimeBetween(result.doclist.docs[0].ctime) }}
+            <hr class="report-divider" align="left" />
+            <div v-if="result.doclist.docs[0].ff_primaryobject_year_from_i && result.doclist.docs[0].ff_primaryobject_year_from_i">
```

Comment:
Jeg tror du har arvet en bug fra mig her - det er 2x det samme object som den kigger om eksisterer. Den sidste burde være ff.primaryobject_year_to_i, istedet for from. Mindre detajle, den ene eksisterer nok ikke uden den anden, men jeg lavede tjekket for at være på den sikre side.

Reply:
Du har ret. Jeg kunne nemlig se, at der var lavet changes i den gamle SingleSearchResult som jeg så havde slettet på min branch, men jeg havde svært ved at pin pointe hvad det var fordi det var en filsletning der kom ind i mit merge.